src/models/mcawsModels.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const Sequelize = require('sequelize');
```

Comment:
Are we planning to use both waterline and sequelize models?

Reply:
its independent from sails-api and  sequelize  is more easy and popular than waterline